packages/flame/lib/src/components/mixins/draggable.dart

```diff
@@ -8,18 +8,37 @@ mixin Draggable on Component {
   bool _isDragged = false;
   bool get isDragged => _isDragged;
 
+  /// Override this to handle the start of a drag/pan gesture that is within the
+  /// boundaries (or hitboxes) of the component that this mixin is used on.
+  /// Return `true` if you want this event to continue to be passed on to
+  /// components underneath (lower priority) this component.
```

Comment:
What's the use case for returning `true` here? Do you want to be able to drag multiple components with a single finger?

Reply:
Either that, or just know that there is a drag event happening on top, you don't have to react to by moving the component.

Reply:
Not sure I understand your example -- this is an `onDragStart` event, which means there can't be any other "drag event happening on top". Unless you mean a drag from another finger, but that sounds like an advanced use case that we defer to `handleDragStart`?